internal/driver/driver.go

```diff
@@ -713,6 +712,13 @@ func getAddr(protocols protocolMap) (net.Addr, error) {
 }
 
 func (d *Driver) addProvisionWatchers() error {
+
+	provisionWatcherFolder := d.config.AppCustom.ProvisionWatcherDir
```

Comment:
This seems to be a missing feature in the SDK similar to `ProfilesDir` and `DevicesDir`. Please submit issue against the SDK (DS will need access to the setting) and add a TODO here to remove this once added to the SDK.

Reply:
Comment added - and issue at https://github.com/edgexfoundry/device-sdk-go/issues/1100

Reply:
Excellent. Thanks!